pkg/store/container/container_test.go

```diff
@@ -69,6 +68,18 @@ func TestContainerStore(t *testing.T) {
 			ImageRef: "TestImage-3",
 			LogPath:  "/test/log/path/3",
 		},
+		"4abcd": {
```

Comment:
nit: Can we change the test metadata above to a long id, too? Feel like it's better to have one more test case.

Reply:
done